modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1,35 +1,31 @@
 = What’s New in Studio
 
-Anypoint Studio 7.13 introduces support for message flow visualization and management of DataWeave libraries through its UI.
+Anypoint Studio 7.14.0 introduces new features to enhance the users' experience making the daily use of Studio easier, faster, and more agile. 
 
-This version also improves the general user experience including new features such as the Exchange Comprehensive Publication.
+== Improved Copy-Paste Experience
 
-== Explore and Debug Shared Flows
+Now, users have the possibility to select if a suffix is added to the copy-pasted Mule Palette elements and to the flows to avoid having similar long names.
 
-Studio now supports message flow visualization of a Mule configuration file loaded from a dependency, which enables exploring and debugging shared flows.
+== Deploy to CloudHub 2.0
 
-image::explore-and-debug-shared-flows.png["A Mule Configuration file inside a dependency is highlighted in the Package Explorer and its flow displayed on the main panel."]
+Aligned with the latest release, Studio allows developers to deploy their applications to https://www.mulesoft.com/platform/saas/cloudhub-ipaas-cloud-based-integration[CloudHub 2.0], which offers a wide variety of shared services.
```

Comment:
Studio allows > Studio enables
Add **^** at the end of the URL to show the link as external: https://www.mulesoft.com/platform/saas/cloudhub-ipaas-cloud-based-integration[CloudHub 2.0^]

Reply:
Thanks for the tip!